scaaml/capture/ascad/picoscope.py

```diff
@@ -0,0 +1,447 @@
+#!/usr/bin/python
```

Comment:
Please add a comment to tell that this files originally comes from Chipwhisperer project. It's not Google code. I think I needed to do some modifications on it or support for it has been dropped when they moved to py3